zavod/zavod/exporters/senzing.py

```diff
@@ -99,7 +99,7 @@ def feed(self, entity: Entity) -> None:
         map(entity, "incorporationDate", record, "DATES", "REGISTRATION_DATE")
         map(entity, "birthPlace", record, "ADDRESSES", "PLACE_OF_BIRTH")
         map(entity, "nationality", record, "COUNTRIES", "NATIONALITY")
-        map(entity, "country", record, "COUNTRIES", "CITIZENSHIP")
+        map(entity, "citizenship", record, "COUNTRIES", "CITIZENSHIP")
```

Comment:
just confirming whether this change can be made yet, and whether we need to map country to another field.

https://senzing.zendesk.com/hc/en-us/article_attachments/30709424434963

Reply:
let's do this:

```
country -> NATIONALITY
citizenship -> CITIZENSHIP
nationality -> NATIONALITY
```